deploy-board/deploy_board/templates/configs/new_capacity.html

```diff
@@ -135,7 +135,7 @@ <h4 class="panel-title pull-left pointer-cursor">Basic Settings</h4>
                 isSelected: item.abstract_name === capacityCreationInfo.defaultSeurityZone
             }
         }),
-        selectedHostTypeValue: capacityCreationInfo.defaultHostType,
+        selectedHostTypeValue: capacityCreationInfo.hostTypes.find(hostType => hostType.abstract_name === capacityCreationInfo.defaultHostType).id,
```

Comment:
can we not use abstract_name here?

Reply:
If I create a host type with same abstract_name as default one, this would fail, correct?

Reply:
How can we get the ID for the defaultHostType?
this option is hardcoded in environment, I don't feel comfortable in putting a default ID on that file.

Reply:
I am not suggesting to use ID. we can use default provider_name when provider is "AWS".